src/main/java/org/sonarqube/gradle/SonarPropertyComputer.java

```diff
@@ -270,7 +273,7 @@ private static void configureJaCoCoCoverageReport(final Test testTask, final boo
   }
 
   private static void configureTestReports(Test testTask, Map<String, Object> properties) {
-    File testResultsDir = testTask.getReports().getJunitXml().getDestination();
+    File testResultsDir = getDestination(testTask.getReports().getJunitXml());
```

Comment:
`getDestination` should be annotated with `CheckForNull` and the following if should check for null.

Reply:
Already done in last commit, or am I missing something?